.github/workflows/multi_service_image_build.yml

```diff
@@ -1,12 +1,13 @@
 ---
-name: multi_service_image_build
+name: multi_build_root
 on:
   push:
-    tags: ['[0-9]+.[0-9]+.[0-9]+-?*']
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+-?*'
 env:
   TAG: ${{ github.ref }}
   REPO_NAME: ${{ github.repository }}
-  ARGO_NAMESPACE: argo
+  ARGO_NAMESPACE: wl-vt-cnask-dev
```

Comment:
Looks like an error here. Need use parameter. Something like that wl-<WL_NAME>-dev

Reply:
Fixed, thank you.